### ISSUE TITLE
Slow inference on static images

### ISSUE THREAD

**Issue by fransay** [ferraridamiano/yolo_flutter#2]
I am looking at a way, we could use your implementation on live object detection.
But looks like inference on static images alone are quite slow. 
Will be learning more about your design and algorithm and see we can improve it.

Great job the way :+1:

**Comment by JobiJoba** [comment]
I would be interested by doing that too but I'm a bit afraid of the slowness of the static image right now.

**Comment by ferraridamiano** [comment]
Yeah, it is true, the current implementation takes really too much time. And I'm not even sure which could be the issue. It could be:
- YOLO (e.g. the model is not well optimized to run on tflite)
- The tflite flutter library (maybe not yet very optimized)
- The app (maybe I did not enabled some option that unlocks more performance)

While the NMS algorithm could be easily speeded up, and maybe I will do it in the near future. But at the moment it takes only a fraction of time of the inference time

**Comment by ctycode** [comment]
> 似乎是 flutter_tflite 的问题：[tensorflow/flutter-tflite#112](https://github.com/tensorflow/flutter-tflite/issues/112)

https://github.com/tensorflow/flutter-tflite/issues/112#issuecomment-2769850318

**Comment by ferraridamiano** [comment]
Fixed by 35d8a10b0bd84a56844a5534c28bcb750a5fb16a
